src/test/scala/temple/utils/StringUtilsTest.scala

```diff
@@ -0,0 +1,22 @@
+package temple.utils
+
+import org.scalatest.{FlatSpec, Matchers}
+import StringUtils.indent
+
+class StringUtilsTest extends FlatSpec with Matchers {
+  "indent" should "add spaces to an empty string" in {
+    indent("") shouldEqual "  "
+  }
+  "indent" should "add n spaces to an empty string" in {
+    indent("", 1) shouldEqual " "
+    indent("", 3) shouldEqual "   "
+  }
+  "indent" should "add spaces to a single line" in {
+    indent("abcd") shouldEqual "  abcd"
```

Comment:
tests will need to be updated if we ever change the default indent size, but I guess we'll never change it